wpe/src/main/java/com/wpe/wpeview/WPEView.java

```diff
@@ -39,6 +39,9 @@ public class WPEView extends FrameLayout implements ViewObserver
     private String m_url = "about:blank";
     private String m_originalUrl = "about:blank";
 
+    private WPESurfaceView m_wpeSurfaceView;
+    private FrameLayout m_customView;
```

Comment:
Between Java and C++ sometimes I've got doubts :laughing: but maybe initialize them to null (or maybe it is redondant here).

Reply:
in java they are initialized to null so let's leave them as they are now